elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHydrator.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.parser.handlebars;
+
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideSecurity;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideTable;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.Table;
+import com.github.jknack.handlebars.Context;
+import com.github.jknack.handlebars.EscapingStrategy;
+import com.github.jknack.handlebars.EscapingStrategy.Hbs;
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Template;
+import com.github.jknack.handlebars.helper.ConditionalHelpers;
+import com.github.jknack.handlebars.io.ClassPathTemplateLoader;
+import com.github.jknack.handlebars.io.TemplateLoader;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class for handlebars hydration.
+ */
+public class HandlebarsHydrator {
+
+    public static final EscapingStrategy MY_ESCAPING_STRATEGY = new Hbs(new String[][]{
+        {"<", "&lt;" },
+        {">", "&gt;" },
+        {"\"", "&quot;" },
+        {"`", "&#x60;" },
+        {"&", "&amp;" }
+    });
+
+    /**
+     * Method to hydrate the Table template.
+     * @param table ElideTable object
+     * @return table java class list
+     * @throws IOException IOException
+     */
+    public List<String> hydrateTableTemplate(ElideTable table) throws IOException {
```

Comment:
ElideTable needs a better class name since it is really a configuration object and not a singular table.